fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java

```diff
@@ -1056,9 +1057,8 @@ public TStatus snapshotLoaderReport(TSnapshotLoaderReportRequest request) throws
     @Override
     public TRefreshTableResponse refreshTable(TRefreshTableRequest request) throws TException {
         try {
-            GlobalStateMgr.getCurrentState().refreshExternalTable(request.getDb_name(),
-                    request.getTable_name(),
-                    request.getPartitions());
+            GlobalStateMgr.getCurrentState().refreshExternalTable(new TableName(request.getCatalog_name(),
```

Comment:
The request.getCatalog_name may be null, because the caller may have not been upgraded. You should make sure it works when this param is null.